common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -197,7 +197,7 @@ def sorting_dates(start, advertised_start, announcement):
         start = dateutil.parser.parse(advertised_start)
         if start.tzinfo is None:
             start = start.replace(tzinfo=utc)
-    except (ValueError, AttributeError):
+    except (TypeError, ValueError, AttributeError):
```

Comment:
@giovannicimolin, `dateutil.parser.parse` in the latest version (2.8.1) only throws `ValueError`. Is `TypeError` also thrown by it or the `.replace()` method?

Reply:
@lgp171188 `TypeError` is thrown by `replace()`, but I think that for this use case the issue will never happen. Is it ok to leave `TypeError` here?

Reply:
@giovannicimolin, no problem. Since you had added it here, I assumed that you knew the reason :)